packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -288,7 +288,7 @@ export async function run(context: $TSContext, resourceDefinition: $TSObject, re
 
         try {
           await updateCloudFormationNestedStack(context, nestedStack, resourcesToBeCreated, resourcesToBeUpdated);
-          if (FeatureFlags.getBoolean('Overrides.project')) {
+          if (FeatureFlags.getBoolean('project.overrides')) {
```

Comment:
is this tied to this change?

Reply:
I changed the name of the FF, but the implementation of it remains unchanged for now. I didn't see the harm in including that in this PR.